docs/reference/data-operations.asciidoc

```diff
@@ -0,0 +1,16 @@
+[[data-operations]]
+= Roll up or transform your data
+
+[partintro]
+--
+
+{es} offers the following methods for manipulating your data:
+
+* <<xpack-rollup,Rolling up your historical data>>
++
+include::rollup/index.asciidoc[tag=rollup-intro]
+* {stack-ov}/ml-dataframes.html[Transforming your data]
+
+--
+
+include::rollup/index.asciidoc[]
```

Comment:
Not blocking, but it does look a little odd to have a short description for rollups, but not transforms.

I understand not wanting to duplicate content or add a dependency to the stack docs. Maybe something to look into later.

<img width="747" alt="transform" src="https://user-images.githubusercontent.com/40268737/65349777-403e5800-dbb2-11e9-9685-d03c852e747b.png">

Reply:
Yes, I'll add that blurb in a subsequent PR when I move the transform content into that section.